DEVELOPMENT.md

```diff
@@ -16,40 +16,81 @@
 
 ## Running Locally
 
-Step 1
```

Comment:
might make sense to move the "dependencies" section above into the "running natively" section, since you won't need those dependencies to go the docker based route :D

Reply:
Reorganized the sections, with "Prerequisites" and "Setup" subsections for docker and native.